test/JDBC/input/BABEL-4271.sql

```diff
@@ -0,0 +1,25 @@
+select 1 where 'ABCD' LIKE 'AB[C]D' ESCAPE ''
```

Comment:
Let's have some tests with table operations as well. For example,
create table t1(a varchar(30), b varchar(30));
insert into t1 values ('abc','abc');

select a,b from t1 where a like b escape '';

Reply:
Done